### ISSUE TITLE
[BE] Response generated by AI is displayed on the frontend

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#32]
Expansion to #30 #25 

## Acceptance criteria
- [ ] The response generated by the AI in the backend is sent to the frontend 
- [ ] The text is displayed on the frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
